### DISCUSSION TITLE
How can I get infrommation from Arch project to use as input of Cicero.

### DISCUSSION THREAD

**Discussion by TauHHCC** [GreenleafLab/ArchR#1875] [Questions / Documentation]
Hi, I'm trying to extract in formation from ArchR project and to use them in Cicero.

My data is established by  plate based method, each cell has been labelled well. 
The data matrix I have now is the fragment file. This file is running fine in ArchR, but I don't think it can be directly used to generate CDS files in cicero since the fragment position is not peaks (actually I have tried, buy my R will die because the file is toooo big)?
Cicero esquires three types information, 1) fragment position, which will be in chr_start_end format; 2) cell name; 3) sequencing counts that overlaps in the fragment position.  Then cicero will take these information to make a CDS file for its analysis. 

My question is, 1) can I extract these information from Arch project? I can find cell name. I can also find  peak position based on pseudo bulk of clusters after peak calling (MACS2), but this peak positions is not for clusters not for every cell. For digging the reads or counts that overlaps in the positions, I have no idea so far. 
2) Can I directly extract the information from Arch project to form a CDS file that can be directly used by the cicero?

Thanks.

**Comment by rcorces** [comment]
I'm not very well versed on the exact input format for Cicero. Are you certain that ArchR's co-accessibility functionality doesnt suit your purpose?

> 1) can I extract these information from Arch project? I can find cell name. I can also find peak position based on pseudo bulk of clusters after peak calling (MACS2), but this peak positions is not for clusters not for every cell. For digging the reads or counts that overlaps in the positions, I have no idea so far.

you should be able to get the information you want using the `getPeakSet()` and `getMatrixFromProject()` functions

> 2) Can I directly extract the information from Arch project to form a CDS file that can be directly used by the cicero?

No. We do not provide exporter functions specifically for this
